factory/usr/lib/systemd/system/run-mnt-base.mount.d/fix-dependencies.conf

```diff
@@ -0,0 +1,4 @@
+# TODO, this should probably be generated by `snap-bootstrap
+# initramfs-mounts`.
```

Comment:
Maybe it would make sense to create an LP bug affecting snapd and ubuntu-core to keep track of this? So we remove it when it gets implemented there.

Reply:
Yes, let's keep this conversation opened until we have a bug opened on launchpad.

Reply:
snapcore/snapd#11263